internal/frontend/default-frontend.go

```diff
@@ -261,6 +261,15 @@ func formatExamples(examples []command.ExampleEntry) string {
 	return strings.Join(output, "\n")
 }
 
+func (self *defaultFrontend) getCommandName(group, name string) string {
+	tokens := []string{self.appCtx.AppName()}
```

Comment:
This appName returned here is hard coded in the main.go file. This is used for the command launcher to get the environment variable PREFIX and determine the default configuration locations: `$HOME/.${appName}`

The code here might not cover a situation in which the user renames the binary into something else. For example, the hard-coded binary is "cl", the user could rename it to "mycl". In this case, the correct value for `CL_FULL_COMMAND_NAME` will be `mycl [group] [name]`.

Reply:
Good point, I change that to use the actual executable name.